tests/functional/u/undefined/undefined_variable.py

```diff
@@ -133,7 +133,7 @@ class Ancestor1(object):
     """ No op """
 
 NANA = BAT # [undefined-variable]
-del BAT
+del BAT  # [undefined-variable]
```

Comment:
This is already warned against on the previous line, isn't it a "false positive" or more accurately a little annoying to have multiple warning for the same problem ?

Reply:
I figured it was the same as the original report in #5399. On consecutive lines it could be a little annoying, but forty lines down it could be informative!

Reply:
Also, it's consistent with the general scenario on `main`:

```
% cat a.py
class A():
  jazz
  jazz


% pylint a.py
************* Module a
a.py:2:2: W0104: Statement seems to have no effect (pointless-statement)
a.py:2:2: E0602: Undefined variable 'jazz' (undefined-variable)
a.py:3:2: W0104: Statement seems to have no effect (pointless-statement)
a.py:3:2: E0602: Undefined variable 'jazz' (undefined-variable)

------------------------------------------------------------------
Your code has been rated at 0.00/10 (previous run: 0.00/10, +0.00)
```

Reply:
> On consecutive lines it could be a little annoying, but forty lines down it could be informative!

Very fair point, I'm resolving